drivers/staging/vc04_services/interface/vchiq_arm/vchiq_core.c

```diff
@@ -891,16 +891,14 @@ queue_message(VCHIQ_STATE_T *state, VCHIQ_SERVICE_T *service,
 						error_count);
 					return VCHIQ_ERROR;
 				}
-				if (i == 0) {
-					if (SRVTRACE_ENABLED(service,
-							VCHIQ_LOG_INFO))
-						vchiq_log_dump_mem("Sent", 0,
-							header->data + pos,
-							min(64u,
-							elements[0].size));
-				}
 			}
 
+		if (SRVTRACE_ENABLED(service,
```

Comment:
* Move the trace message outside the gather loop so fragmentation
doesn't mask significant bytes.
* Reduce the size of the data logged - 16 bytes is sufficient.

Signed-off-by: Phil Elwell <phil@raspberrypi.org>